test/ProductConstructionService.ScenarioTests/ScenarioTests/ScenarioTests_CodeFlow.cs

```diff
@@ -104,48 +108,67 @@ public async Task Vmr_BackwardFlowTest()
             TestParameters.GitHubTestOrg,
             sourceDirectory: TestRepository.TestRepo1Name);
 
-        // Clone the VMR
+        TemporaryDirectory testRepoFolder = await CloneRepositoryAsync(TestRepository.TestRepo1Name);
         TemporaryDirectory reposFolder = await CloneRepositoryAsync(TestRepository.VmrTestRepoName);
         var newFilePath = Path.Combine(reposFolder.Directory, "src", TestRepository.TestRepo1Name, TestFileName);
 
-        using (ChangeDirectory(reposFolder.Directory))
+        await CreateTargetBranchAndExecuteTest(targetBranchName, testRepoFolder, async () =>
```

Comment:
Shouldn't the `targetBranchName` be something dynamic but not `master` basically?

Reply:
I didn't push my local branch where this was fixed, thanks